action.yml

```
@@ -36,14 +36,17 @@ inputs:
   trivy_severity:
     description: "Severities of vulnerabilities to scanned for and displayed."
     default: "CRITICAL,HIGH"
+  ignore_trivy:
+    description: "Whether to ignore Trivy vulnerabilities."
+    default: "false"
   dockerhub_platforms:
       description: "Platforms passed to DockerHub build and push action."
-      default: "linux/amd64,linux/arm64"
+      default: "linux/arm64,linux/arm64"
```

```suggestion
      default: "linux/amd64,linux/arm64"
```
That should stay the same since it is later used for the actual push step, or?